modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonAbstractConnexionServerCodegen.java

```diff
@@ -963,7 +963,9 @@ public void postProcessParameter(CodegenParameter parameter) {
             }
             if (operation.requestBodyExamples != null) {
                 for (Map<String, String> example : operation.requestBodyExamples) {
-                    if (example.get("contentType") != null && example.get("contentType").equals("application/json")) {
+                    if (operation.bodyParam.example == null
+                    		&& example.get("contentType") != null 
+                    		&& example.get("contentType").equals("application/json")) {
```

Comment:
FYI. You can do `"application/json".equals(example.get("contentType"))` instead to avoid the null check.

Reply:
Good point.
It was a part of code done here before me, I just add line breaks.

My changes related to the issue - check whether `operation.bodyParam.example` is not set, and do nothing if it is already there